src/main/target/OMNIBUSF4/target.h

```diff
@@ -172,10 +172,17 @@
 #define ENABLE_BLACKBOX_LOGGING_ON_SPIFLASH_BY_DEFAULT
 #define FLASH_CS_PIN            SPI3_NSS_PIN
 #define FLASH_SPI_INSTANCE      SPI3
-#define USE_FLASHFS
```

Comment:
Is the exposed CS pin on this board shared with the flash chip? That's a bit unfortunate, as we should not disable the flash functionality.

Reply:
These two lines are defined twice, so I removed second occurence as housekeeping.
Line 146 and 147 in config.h.